packages/ozone/src/mod-service/index.ts

```diff
@@ -181,6 +185,24 @@ export class ModerationService {
           .if(!subjectUri, (q) => q.where('subjectUri', 'is', null))
           .if(!!subjectUri, (q) => q.where('subjectUri', '=', subjectUri))
       }
+    } else if (subjectType === 'account') {
+      builder = builder.where('subjectUri', 'is', null)
+    } else if (subjectType === 'record') {
+      builder = builder.where('subjectUri', 'is not', null)
+    }
+
+    // If subjectType is set to 'account' let that take priority and ignore collections filter
+    if (collections.length && subjectType !== 'account') {
+      builder = builder.where('subjectUri', 'is not', null).where((qb) => {
+        collections.forEach((collection, index) => {
+          if (index === 0) {
+            qb = qb.where('subjectUri', 'like', `%/${collection}/%`)
+          } else {
+            qb = qb.orWhere('subjectUri', 'like', `%/${collection}/%`)
+          }
+        })
```

Comment:
Pretty sure this should work just fine!
```suggestion
        collections.forEach((collection) => {
          qb = qb.orWhere('subjectUri', 'like', `%/${collection}/%`)
        })
```